stories/qa/data-table/data-table.stories.html

```diff
@@ -889,6 +885,16 @@
 						</tbody>
 					</table>
 				</div>
+				<tr class="dataTable-body-row mod-draggable cdk-drag-preview" aria-hidden="true" style="inline-size: 937px; block-size: 44px">
+					<td class="dataTable-body-row-cell" aria-hidden="true">
```

Comment:
Redundant with its parent having one already?

```suggestion
					<td class="dataTable-body-row-cell">
```